src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -195,19 +242,25 @@ public void driveFieldRelative(ChassisSpeeds speeds) {
   }
 
   public SwerveModulePosition[] getModulePositions() {
-    getModuleStates();
+    getModuleStates(); // TODO: REMOVE IF NOT NEEDED! It's returning a 
```

Comment:
argh, that's a dumb hack that makes sure we log the states every periodic(). should just move that call into periodic() explicitly with a comment that its just being called for logging purposes. akit's `@AutoLog` annotation might work too but i'd rather convert everything to use that in one go.

Reply:
Let's make an issue for this